src/modules/package.rs

```diff
@@ -102,9 +102,24 @@ fn get_setup_cfg_version(context: &Context, config: &PackageConfig) -> Option<St
 }
 
 fn get_gradle_version(context: &Context, config: &PackageConfig) -> Option<String> {
-    let file_contents = context.read_file_from_pwd("build.gradle")?;
-    let re = Regex::new(r#"(?m)^version ['"](?P<version>[^'"]+)['"]$"#).unwrap();
-    let caps = re.captures(&file_contents)?;
+    let build_file_contents = match context.read_file_from_pwd("build.gradle") {
+        Some(contents) => contents,
+        None => "none".to_string(),
+    };
+
+    //check version from gradle.properties file
+    if build_file_contents == "none" {
+        let properties_file_contents = context.read_file_from_pwd("gradle.properties")?;
+        let re = Regex::new(r"/version=.*/gm").unwrap();
+        let caps = re.captures(&properties_file_contents)?;
+        return format_version(
+            &caps["version"].strip_prefix("version=").unwrap(),
```

Comment:
Please either use proper capture groups in the regex or just iterate over the lines and check if `strip_prefix` returns `Some(version)`. Please also add handling for whitespace if necessary.

Reply:
Capture groups was easier.